engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -400,7 +404,7 @@ public ClassLoader getExternalClassLoader() {
 
 	@Override
 	public final MessageInterpolator getDefaultMessageInterpolator() {
-		return defaultMessageInterpolator;
+		return new ResourceBundleMessageInterpolator( defaultResourceBundleLocator );
```

Comment:
Just to be clear on my worries: it's named like a getter and it used to work like a getter so I didn't think it was such a trivial change to make it return a different instance for each call. I don't think it's an inappropriate expectation to have it behave like a getter.